products/memgraph/upcoming/reference_guide/query-modules.md

```diff
@@ -98,6 +98,17 @@ offering.
 * `mg`: Utility module that offers more insight into custom query modules.
 * `louvain` [Enterprise]: Louvain algorithm for community detection.
 * `connectivity` [Enterprise]: Algorithms that analyse graph connectivity.
+* `nxalg` : Wrapper functions providing the most common algorithms found
+            in the NetworkX package. The users can extend the module with
+            their own custom functions based on NetworkX algorithms.
+
+NOTE:
+If you intend to implement your custom algorithms based on the NetworkX package,
+please note that we provide the extra library `mgp_networkx.py' module
```

Comment:
Replace ' with ` :)

Reply:
I'm not following :) Could you be more specific?

Reply:
I think your goal was to put mgp_network.py to monospace? That's not the case because there is ' at the end of the filename. Under `Files changes`, there is `Display rich diff` functionality where you can see a rendered version of the markdown.